bundles/org.openhab.binding.radiothermostat/src/main/java/org/openhab/binding/radiothermostat/internal/handler/RadioThermostatHandler.java

```diff
@@ -235,22 +235,27 @@ private void scheduleClockSyncJob() {
      * Sync the thermostat's clock with the host system clock
      */
     private void syncThermostatClock() {
-        Calendar c = Calendar.getInstance();
+        boolean success = false;
+
+        // don't sync clock if override is on because it will reset temporary hold
+        if (rthermData.getThermostatData().getOverride() == null
+                || rthermData.getThermostatData().getOverride().compareTo(0) == 0) {
+            Calendar c = Calendar.getInstance();
+
+            // The thermostat week starts as Monday = 0, subtract 2 since in standard DoW Monday = 2
+            int thermDayOfWeek = c.get(Calendar.DAY_OF_WEEK) - 2;
+            // Sunday will be -1, so add 7 to make it 6
+            if (thermDayOfWeek < 0) {
+                thermDayOfWeek += 7;
+            }
 
-        // The thermostat week starts as Monday = 0, subtract 2 since in standard DoW Monday = 2
-        int thermDayOfWeek = c.get(Calendar.DAY_OF_WEEK) - 2;
-        // Sunday will be -1, so add 7 to make it 6
-        if (thermDayOfWeek < 0) {
-            thermDayOfWeek += 7;
+            success = connector.sendCommand(null, null,
+                    String.format(JSON_TIME, thermDayOfWeek, c.get(Calendar.HOUR_OF_DAY), c.get(Calendar.MINUTE)),
+                    TIME_RESOURCE).contains("success");
         }
 
-        final String response = connector.sendCommand(null, null,
-                String.format(JSON_TIME, thermDayOfWeek, c.get(Calendar.HOUR_OF_DAY), c.get(Calendar.MINUTE)),
-                TIME_RESOURCE);
-
         // if sync call was successful run again in one hour, if un-successful try again in one minute
-        this.clockSyncJob = scheduler.schedule(this::syncThermostatClock, (response.contains("success") ? 60 : 1),
-                TimeUnit.MINUTES);
+        this.clockSyncJob = scheduler.schedule(this::syncThermostatClock, (success ? 60 : 1), TimeUnit.MINUTES);
```

Comment:
Should it try again in one minute when having override?

Reply:
I though that it would be be better so that the sync will run within a few minutes after the override expires. Otherwise it will be up to an hour before the sync is called again. Unless you think that it is too expensive to have the this additional job scheduler run every minute. WDYT?

Reply:
I was not sure how the override value is updated, so if it's just the cost of a periodic job to have it synced faster, sure, it's probably worth it. Just to be sure: The override is not polled by this method (line 241), right? In this case it would probably be too expensive - not for openHAB, but for the thermostat if it runs on batteries. Syncing the clock is probably not "important enough" to be done so frequently in the case where override is on?